Pipfile

```diff
@@ -20,17 +20,27 @@ django-imagekit = "*"
 django-storages = "*"
 django-multiselectfield = "*"
 boto3 = "*"
-sentry-sdk = {extras = ["django"], version = "*"}
+sentry-sdk = {extras = ["django"] }
```

Comment:
Why is this changing?

Reply:
I'm not sure. I'll run `pipenv install` again and see if that changes anything.

Reply:
I've reverted it.